examples/mwaa/main.tf

```diff
@@ -0,0 +1,317 @@
+#------------------------------------------------------------------------
+module "aws_vpc" {
+  source  = "terraform-aws-modules/vpc/aws"
+  version = "~> 3.0"
+
+  name = local.vpc_name
+  cidr = local.vpc_cidr
+  azs  = local.azs
+
+  public_subnets  = [for k, v in local.azs : cidrsubnet(local.vpc_cidr, 8, k)]
+  private_subnets = [for k, v in local.azs : cidrsubnet(local.vpc_cidr, 8, k + 10)]
+
+  enable_nat_gateway   = true
+  create_igw           = true
+  enable_dns_hostnames = true
+  single_nat_gateway   = true
+
+  public_subnet_tags = {
+    "kubernetes.io/cluster/${local.cluster_name}" = "shared"
+    "kubernetes.io/role/elb"                      = "1"
+  }
+
+  private_subnet_tags = {
+    "kubernetes.io/cluster/${local.cluster_name}" = "shared"
+    "kubernetes.io/role/internal-elb"             = "1"
+  }
+}
+
+#------------------------------------------------------------------------
+# AWS EKS Blueprints Module
+#------------------------------------------------------------------------
+module "eks_blueprints" {
+  source = "../.."
+
+  tenant            = local.tenant
+  environment       = local.environment
+  zone              = local.zone
+  terraform_version = local.terraform_version
+
+  # EKS Cluster VPC and Subnet mandatory config
+  vpc_id             = module.aws_vpc.vpc_id
+  private_subnet_ids = module.aws_vpc.private_subnets
+
+  # Attach additional security group ids to Worker Security group ID
+  worker_additional_security_group_ids = [] # Optional
+
+  # EKS CONTROL PLANE VARIABLES
+  cluster_version = local.cluster_version
+
+  # Add MWAA IAM Role to aws-auth configmap
+  map_roles = [
+    {
+      rolearn  = "${module.mwaa.mwaa_role_arn}" # The ARN of the IAM role
+      username = "mwaa-role"                    # The user name within Kubernetes to map to the IAM role
+      groups   = ["system:masters"]             # A list of groups within Kubernetes to which the role is mapped; Checkout K8s Role and Rolebindings
+    }
+  ]
+
+  # EKS MANAGED NODE GROUPS
+  managed_node_groups = {
+    # Managed Node groups with minimum config
+    mg5 = {
+      node_group_name = "mg5"
+      instance_types  = ["m5.large"]
+      min_size        = "2"
+      disk_size       = 100 # Disk size is used only with Managed Node Groups without Launch Templates
+    }
+  }
+}
+
+#------------------------------------------------------------------------
+# Kubernetes Add-on Module
+#------------------------------------------------------------------------
+module "eks_blueprints_kubernetes_addons" {
+  source = "../../modules/kubernetes-addons"
+
+  eks_cluster_id = module.eks_blueprints.eks_cluster_id
+
+  enable_metrics_server     = false
+  enable_cluster_autoscaler = false
+
+}
+
+#------------------------------------------------------------------------
+# AWS MWAA Module
+#------------------------------------------------------------------------
+
+module "mwaa" {
+  source                        = "aws-ia/mwaa/aws"
```

Comment:
@vara-bonthu I'm pointing to aws-ia/mwaa/aws but we haven't published the module there yet, should I create a release on the github module instead and point to the release tag?

Reply:
We can merge both the MWAA module and this example at the same time.